api/application.go

```diff
@@ -1112,13 +1069,15 @@ type Application struct {
 	Comments        string      `json:"comments"`
 	Identities      []Ref       `json:"identities"`
 	Tags            []TagRef    `json:"tags"`
-	BusinessService *Ref        `json:"businessService"`
+	BusinessService *Ref        `json:"businessService" yaml:"businessService"`
```

Comment:
Did these while I was here, still need to do a comprehensive sweep in another PR.